pages/index.tsx

```diff
@@ -12,49 +10,55 @@ import React, { useState } from 'react';
  * @version 1.0.0
  */
 const Home: NextPage = (): JSX.Element => {
-  // const [name, setName] = useState('jojo');
   const [isOn, setIsOn] = useState<boolean>(false);
-  // const [isYou, setIsYou] = useState<string>('jojo');
-
-  // const onSignUp = (e: FormEvent<HTMLFormElement>) => {
-  //   e.preventDefault();
-  //   console.log(name);
-  //   setName('Kimkya');
-  //   console.log(name);
-  // };
-
+  const [isActiveted, setIsActiveted] = useState<boolean>(false);
```

Comment:
can you change isActiveted to isActivated???

Reply:
ohh! yeah sure, I did not pay attention on the word.. thanks